README.md

```diff
@@ -1,2 +1,82 @@
-# tpcc
-Repo to run TPCC benchmarks against YugabyteDB
+# TPCC benchmark
+
+This is a fork of Oltp bench that is used to run the TPCC benchmark. All the other benchmarks have been removed.
+
+
+Just like the original bencmark this is a multi-threaded load generator. The framework is designed to be able to produce variable rate,
+variable mixture load against any JDBC-enabled relational database. The framework also provides data collection
+features, e.g., per-transaction-type latency and throughput logs.
+
+## Dependencies
+
++ Java (+1.7)
++ Apache Ant
+
+
+## Environment Setup
++ Install Java and Ant.
++ Download the source code.
+  ```bash
+  git clone https://github.com/yugabyte/tpcc.git
+  ```
++ Run the following commands to build:
+  ```bash
+  ant bootstrap
+  ant resolve
+  ant build
+  ```
+
+## Setup of the Database
+If you start from scratch, you should create an empty database (e.g., for TPC-C you can create a db named <b>tpcc</b>) and provide login credential to the benchmark, by modifying  <b>the workload descriptor file</b>. The ./config directory provides several examples, we now use the sample_tpcc_config.xml. You should edit the following portion:
```

Comment:
Please provide ysql commands that users will need to run for this.

Reply:
There are no ysql commands necessary.
IT creates the DB/table and inserts into the table in the load step.